polytope/polytope.py

```diff
@@ -570,6 +564,18 @@ def _rotate(polyreg, i=None, j=None, u=None, v=None, theta=None, R=None):
     return R
 
 
+def _Givens_rotation_matrix(i, j, theta, N):
```

Comment:
I agree with splitting the file into topic-oriented files. It should be possible to do this and avoid troubles with cycle dependencies. However, I think that it is outside the scope of this PR.